src/lib/auth/AuthProviders/OAuthAuthenticator/index.ts

```diff
@@ -0,0 +1,122 @@
+import axios from 'axios'
+import { createHash } from 'crypto'
+import isWsl from 'is-wsl'
+import jwt from 'jsonwebtoken'
+import opn from 'opn'
```

Comment:
This package has been deprecated and renamed to [open](https://www.npmjs.com/package/open).
I think it would be better if we updated it! :)

Reply:
Actually I tried some months ago to use the new package, but we had some issues with some systems - WSL - I made a contribution to fix the issue https://github.com/sindresorhus/open/pull/171 - however we began having issues with other systems and some specific accounts URLs :(
Because of this I decided to wait for the 'open' package to evolve more and then update